src/three-components/SmoothControls.ts

```diff
@@ -597,4 +651,41 @@ export class SmoothControls extends EventDispatcher {
       event.preventDefault();
     }
   }
+
+  [$calculateOverSphericals]() {
+    const {
+      minimumAzimuthalAngle,
+      maximumAzimuthalAngle,
+      minimumPolarAngle,
+      maximumPolarAngle,
+      minimumRadius,
+      maximumRadius,
+      overRotateAmount,
+    } = this[$options];
+
+    const minOverRotatePolar = (0 - minimumPolarAngle!) * overRotateAmount! + minimumPolarAngle!;
+    const maxOverRotatePolar = (Math.PI - maximumPolarAngle!) * overRotateAmount! + maximumPolarAngle!;
+    let minOverRotateAzimuthal = minimumAzimuthalAngle!;
+    let maxOverRotateAzimuthal = maximumAzimuthalAngle!; 
+
+    if (minimumAzimuthalAngle! > -Math.PI) {
+      minOverRotateAzimuthal = (minimumAzimuthalAngle! - Math.PI) * overRotateAmount! + minimumAzimuthalAngle!;
+    }
+
+    if (maximumAzimuthalAngle! < Math.PI) {
+      maxOverRotateAzimuthal = (Math.PI - maximumAzimuthalAngle!) * overRotateAmount! + maximumAzimuthalAngle!;
+    }
```

Comment:
Azimuth angles are a bit weird with `overRotateAmount`. This might be easier if we just :fire: `overRotateAmount` and have a distinct over amount to rotate for azimuth. Not sure if it should be the same for polar angles.

With the above if `maximumAzimuthalAngle` is greater than PI no "elastic" effect is applied.